MainWindow.xaml

```diff
@@ -5,56 +5,92 @@
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height=".25*"/>
+            <RowDefinition Height="120"></RowDefinition>
+            <RowDefinition Height="*"></RowDefinition>
+            <RowDefinition Height="60"></RowDefinition>
         </Grid.RowDefinitions>
         <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="0.25*"/>
-            <ColumnDefinition Width="*"/>
-            <ColumnDefinition Width="*"/>
-            <ColumnDefinition Width="*"/>
-            <ColumnDefinition Width="*"/>
-            <ColumnDefinition Width="*"/>
-            <ColumnDefinition Width="*"/>
-            <ColumnDefinition Width="0.25*"/>
+            <ColumnDefinition Width="180"></ColumnDefinition>
+            <ColumnDefinition Width="*"></ColumnDefinition>
         </Grid.ColumnDefinitions>
-        <Grid Grid.ColumnSpan="8" Grid.RowSpan="2"/>
+        
+        <Rectangle Fill="#1b1d26" Grid.Column="0" Grid.RowSpan="3" Grid.Row="0"></Rectangle>
+        <Image Margin="12,8,12,8" Source="Images/icon.png" Stretch="Uniform"></Image>
+        
+        <!-- Title and version -->
+        <StackPanel Grid.Row="1" Grid.Column="0">
+            <TextBlock
+            Padding="0"
+            FontSize="22"
+            FontWeight="Light"
+            TextWrapping="Wrap"
+            LineHeight="26"
+            LineStackingStrategy="BlockLineHeight"
+            TextAlignment="Center"
+            Foreground="{StaticResource ForegroundCoralBrush}"
+            FontFamily="./Fonts/#Poppins"
+            Grid.Column="0"
+            Grid.Row="1">Shell City<LineBreak/>Mod Loader</TextBlock>
+            
+            <TextBlock 
+                Grid.Row="1" 
+                HorizontalAlignment="Center"
+                Foreground="{StaticResource ForegroundWhiteImportantBrush}"
+                  FontFamily="./Fonts/#Poppins" TextAlignment="Center" Grid.RowSpan="2">v0.1</TextBlock>
+        </StackPanel>
+        
+        <!-- Developer info -->
+        <TextBlock
+            Foreground="{StaticResource ForegroundWhiteStandardBrush}" 
+            FontFamily="./Fonts/#Poppins"
+            Grid.Column="0"
+            Grid.Row="2"
+            HorizontalAlignment="Center" TextAlignment="Center">
+            Developed by Suprnova<LineBreak/>View on <Hyperlink TextDecorations="None" Foreground="{StaticResource ForegroundPurpleBrush}" NavigateUri="https://github.com/Suprnova/Shell-City-Mod-Loader">GitHub</Hyperlink></TextBlock>
```

Comment:
do me a favor and add your name here too for ui design bestie

Reply:
babe 😳😳😳